crates/goose-server/src/configuration.rs

```diff
@@ -88,6 +89,17 @@ pub enum ProviderSettings {
         #[serde(default)]
         max_tokens: Option<i32>,
     },
+    Groq {
```

Comment:
one thing I missed in the google provider and here initially was that we also want to add context_limit and estimate_factor here as well